.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -156,6 +156,8 @@ jobs:
           
           if [[ "${{ steps.set-vars.outputs.pkg_type }}" != "none" ]]; then
             sudo apt-get -qq install -y doxygen ghostscript graphviz
+            echo "Testing doxygen"
+            doxygen --version
```

Comment:
This seems to (somehow) make the Mac runner also install Doxygen (which then fails). I cannot for the life of me figure out how, since this is in a block the only gets run on Ubuntu.

Reply:
I think macOS has its own doxygen installation command which gets run. If you don't want to install it on mac, you should remove it from there.

Reply:
There was just an update to one of the brew formulas that broke everything. I expect it to break in the nightlies today as well.